content/OTP/Guides/OTP_Walk-through.adoc

```diff
@@ -93,13 +94,14 @@ The YubiKey connects to the api.yubico.com validation server. The browser displa
 
 a) Click Continue.
 
-b) Tap YubiKey. The YubiKey in this case is the keyboard
+b) Tap the YubiKey. The YubiKey in this case is the keyboard.
 
 c) Select Keyboard Type. Then click Done.
 
-d) Tap YubiKey again.
+d) Tap the YubiKey again.
+
+The browser page updates and completes an OTP. The first twelve modhex characters of the `opt=` are the Private Key for the inserted YubiKey.
```

Comment:
Chris Halos: The first twelve modhex characters of the `opt=` ... shouldn't this be otp?